src/components/DatabaseList/DatabaseList.module.css

```diff
@@ -92,6 +92,7 @@
   flex-grow: 1;
   display: flex;
   font-size: 16px;
+  padding-top: 15%;
```

Comment:
we shall have to sync on this

Reply:
We had a sync on this and the problem was that the container didn't have any height at all and we gave it that.

Reply:
We had a sync on this and the problem was that the container didn't have any height at all and we gave it that.